---
layout: page
title: The Gut Microbiome in Autism
body_class: projects
published: true
---

# The Gut Microbiome in Autism
<hr>

## Overview
The existence of a link between the gut and autism is well established, yet mechanisms connecting the two remain poorly understood (Konstantynowicz et al., Er. J. Paediatr. Neurol 2012, Hsiao et al., Cell. 1023). A complete mapping of the microbial diversity across the autism spectrum could result in dramatic clinical and therapeutic advances. We have initiated a plan to sequence and analyze the microbiomes of young children with autism (2-7 year old) and age matched unaffected siblings (within 2 years). Leveraging social networks, our goal is to conduct a completely crowdsourced clinical trial that will enroll hundreds of internet-active families with autism quickly – in months instead of years. Participants submitted noninvasively collected samples from home as well as 2-5 min video of their child, giving us unprecedentedly broad 16S amplicon data, allowing us to search for a microbial community specific to ASD. In addition to looking at the microbial structure of the samples, we’re collecting saliva samples to genotype the participants. This study aims to improve our understanding of the link between microbiome functionality, genome variation, and ASD phenotype, and reveal the specific mechanisms by which the gut microbiome interacts with autism-related alleles to produce and modify ASD.

## Recent Work

Human models have demonstrated limited success in identifying gut microbial signatures of diseases such as autism, diabetes, and obesity. While 16s sequencing can identify the relative abundance of specific microbes within an individual's gut, the downstream analysis of 16s data often relies on considering each unique sequence as a different microbe or querying a database to get taxonomic labels for sequences.  We hypothesize that neither of these approaches fully captures phylogenetic relationships between microbes,  consequently weakening the signal distinguishing differentially abundant groups of taxa between a diseased and control cohort. To better capture phylogenetic relationships between taxa, we present a method that groups and aggregates taxa counts across 16s sequence-based biomarkers (SBBs) using single loci variants and combinations of loci variants within the 16s sequences. In a dataset of 115 pairs of children with autism and their neurotypical siblings, we show that using SBBs, we can identify several differentially abundant microbial groups, primarily composed of Veillonellaceae and Porphyromonadaceae (FDR < 0.1). We validate this method on controlled datasets of obese adults and of Type 1 diabetics.  Using SBBs, we find several strongly differentially abundant microbial groups in obesity, primarily groups composed of Gastranaerophilaceae, Megasphaeraceae and Burkholderiaceae, and Oscillospiraceae (FDR < 0.05).  In Type 1 diabetes, we find no differentially abundant taxa using traditional analysis or our method, adding to the withstanding mystery of the gut microbiome's role in diabetes.  In both the autism and obesity datasets, SBBs achieve much higher statistical power and identify differentially abundant groups of microbes undetectable using traditional taxonomic aggregation.  These results pave a new avenue for harnessing the power of 16s sequencing to understand microbial signatures of complex human disease.



[IEEE Publication](https://ieeexplore.ieee.org/abstract/document/8621554)

[KQED Publication](https://ww2.kqed.org/futureofyou/2016/10/04/microbiome_autism/)

[SF Chronicle Press Release](http://www.sfgate.com/business/article/Researchers-explore-link-between-gut-bacteria-and-12250299.php)