---
layout: alumnus
group: alumnus_new
position: Assistant Professor, Dept. of Microbiology, Oregon State University
first_name: Maude
last_name: David
email_prefix: maudemdavid
email_suffix: gmail, com
website: https://microbiology.science.oregonstate.edu/maude-david
phone:
fax:
picture: maude.png
---

Maude joined the lab in January 2014 and namely studies the gut microbiome of children with ASD (Autism Spectrum Disorder) utilizing a large scale, crowd sourced clinical study approach. Her expertise are in microbiology, bioinformatics and genomics, using and integrating metagenomics, metatranscriptomics and metaproteomics to understand microbial community functions.
She received her PhD in February 2010 from the Ecole Centrale de Lyon, University of Lyon, France, with Prof. T.M. Vogel, on the origin of the dehalogenases and bioremediation of chlorinated solvent.  Her grad-school work focused on the bacterial adaptation to chlorinated compounds at the genome (evolution mechanisms) and community (bioremediation) level. After graduation, she became a post-doctoral fellow at Lawrence Berkeley National Laboratory with Prof. Janet Jansson. Her work looked at the impact of climate change on soil microbial ecology and specifically at how altered precipitation affect carbon cycle using meta-“omics” analysis of microbial carbon cycling responses.

### Press
[KQED: Researchers Hunt for a Link Between Microbiome and Autism by Danielle Venton](https://ww2.kqed.org/futureofyou/2016/10/04/microbiome_autism/)

[CEHG: Cross-referencing analysis of autism to identify novel genes and pathways by Pleuni Pennings](https://evolgenomecehg.wordpress.com/2014/06/07/june-11th-maude-david-cross-disorder-analysis-of-autism-to-identify-novel-genes-and-pathways/)

### Teaching Material
[Science in the classroom: annotated research paper for use in university - KC Hokanson & MM David](http://scienceintheclassroom.org/research-papers/you-are-what-you-eatat-least-your-brain/university) - “The gut microbiota influences blood-brain barrier permeability in mice” Braniste et al, Science Magazine, 2016

### Peer-Reviewed Publications
<b> David MM </b>, BA Babineau, DP Wall. Can we accelerate autism discoveries through crowdsourcing? RASD. 2016. 32:80-83. [http://dx.doi.org/10.1016/j.rasd.2016.09.001](http://dx.doi.org/10.1016/j.rasd.2016.09.001)

<b> David MM </b>, D Enard, A Ozturk, J Daniels, JY Jung, L Diaz-Beltran, DP. Wall. Comorbid Analysis of Genes Associated with Autism Spectrum Disorders Reveals Differential Evolutionary Constraints. PLoS ONE. 2016. 11(7):e0157937. [http://dx.doi.org/10.1371/journal.pone.0157937](http://dx.doi.org/10.1371/journal.pone.0157937)

J Hultman, MP Waldrop, R Mackelprang, <b> David MM </b>, J McFarland, SJ Blazewicz, J Harden, M  Turetsky, AD McGuire, MB Shah, NC VerBerkmoes, LH Lee, K Mavrommatis, JK Jansson.  Multi-omics of permafrost, active layer and thermokarst bog soil microbiomes. Nature. 2015. 521:208-212. [http://dx.doi.org/10.1038/nature14238](http://dx.doi.org/10.1038/nature14238)

E Prestat *& <b> David MM *</b>, J Hultman, N Taş, R Lamendella, J Dvornik, R Mackelprang, D Myrold, A. Jumpponen, S. Tringe, E. Holman, K. Mavromatis, JK. Jansson. Functional Ontology Assignments for Metagenomes: a Hidden Markov Model (HMM) database with environmental focus. Nucleic Acids Res. 2014. 42(19):e145. [http://dx.doi.org/10.1093/nar/gku702](http://dx.doi.org/10.1093/nar/gku702) *the first two authors should be regarded as joint First Authors

<b> David MM </b>, S Cecillon, BM Warne, E Prestat, JK Jansson, TM Vogel. Microbial ecology of chlorinated solvent biodegradation. Environ. Microbiol. 2014. Dec;17(12):4835-50. [http://dx.doi.org/10.1111/1462-2920.12413](http://dx.doi.org/10.1111/1462-2920.12413)

D Bravo, M Gaëtan, <b> David MM </b>, G Cailleu, E Verrecchia, P Junier. Identification of active oxalotrophic bacteria by Bromodeoxyuridine DNA-labeling in a microcosm soil experiments. FEMS Microbiol Lett. 2013. 348(2):103–111. [http://dx.doi.org/10.1111/1574-6968.12244](http://dx.doi.org/10.1111/1574-6968.12244)

CD Nicora, BJ Anderson, SJ Callister, AD Norbeck, SO Purvine, JK Jansson, OU Mason, <b> David MM </b>, D Jurelevicius, RD Smith, MS Lipton. Amino acid treatment enhances protein recovery from sediment and soils for metaproteomic studies. Proteomics. 2013. 13(18-19):2776-85. [http://dx.doi.org/10.1002/pmic.201300003](http://dx.doi.org/10.1002/pmic.201300003)

J Bælum, E Prestat, <b> David MM </b>, B Strobel, and C Jacobsen.  Modelling phenoxy acid herbicide mineralization and growth of microbial degraders in 15 soils monitored by quantitative real-time PCR of the functional tfdA gene. Appl. Environ. Microbiol. 2012. 78(15):5305-12.  [http://dx.doi.org/10.1128/AEM.00990-12](http://dx.doi.org/10.1128/AEM.00990-12)

E Dugat-Bony, C Biderre-Petit, F Jaziri, <b> David MM </b>, J Denonfoux, DY Lyon, JY Richard, C Curvers, D Boucher, TM Vogel, E Peyretaillade, P Peyret. In situ TCE degradation mediated by complex dehalorespiring communities during biostimulation processes. J Microbial Biotech. 2012. 5(5):642-53.  [http://dx.doi.org/10.1111/j.1751-7915.2012.00339.x](http://dx.doi.org/10.1111/j.1751-7915.2012.00339.x)

R Mackelprang, MP Waldrop, KM Deangelis, <b> David MM </b>, KL Chavarria, SJ Blazewicz, EM Rubin and JK Jansson. Metagenomic analysis of a permafrost microbial community reveals a rapid response to thaw. Nature. 2011.480(7377):368-71. [http://dx.doi.org/10.1038/nature10576](http://dx.doi.org/10.1038/nature10576)

A Nemir, <b> David MM </b>, R Perrussel, A Sapkota, P Simonet, JM Monier and TM Vogel. Comparative phylogenetic microarray analysis of microbial communities in TCE-contaminated soils. Chemosphere. 2010. (80):600-607. [http://dx.doi.org/10.1016/j.chemosphere.2010.03.036](http://dx.doi.org/10.1016/j.chemosphere.2010.03.036)

S Demaneche, L Philippot, <b> David MM </b>, E Navarro, TM Vogel, and P Simonet. Characterization of Denitrification Gene Clusters of Soil Bacteria via a Metagenomic Approach.  Appl. Environ. Microbiol. 2009.75(2):534–537. [http://dx.doi.org/10.1128/AEM.01706-08](http://dx.doi.org/10.1128/AEM.01706-08)

S Demaneche, <b> David MM </b>, E Navarro, P Simonet and TM Vogel. Evaluation of functional gene enrichment in a soil metagenomic clone library. J Microbiol Meth. 2009.76(1):105-7. [http://dx.doi.org/10.1016/j.mimet.2008.09.009](http://dx.doi.org/10.1016/j.mimet.2008.09.009)

<b> David MM </b>, AR Sapkota, P Simonet and TM Vogel. A novel and rapid method for synthesizing positive controls and standards for quantitative PCR.  J Microbiol Meth. Methods. 2008. 73(1):73–7.  [http://dx.doi.org/10.1016/j.mimet.2008.01.001](http://dx.doi.org/10.1016/j.mimet.2008.01.001)

<b> David MM </b>, F Bringel, M Pagni, N Gilmartin, H Boubakri, T Nadalig, P Simonet, TM Vogel and S Vuillemier. Diversité et évolution des déshalogénases bactériennes : détection bioinformatique et perspectives de recherches (Diversity and evolution of bacterial dehalogenases: bioinformatic detection and research prospects). Les Actes du BRG. 2008. 7:83- 9

### Book Chapters
Berger S, <b> David MM </b>, Vogel TM, Simonet P. Chapter 16 : Rôle du transfert horizontal de gènes dans la dégradation biologique des molécules chlorées : exemple de la dégradation biologique du lindane. In: Écologie microbienne: microbiologie des milieux naturels et anthropisés (Microbial Ecology : microbiology of natural and anthropic environment).  Presses Universitaires de Pau.

### Proceedings

Sochat V, <b>David MM</b>, DP Wall. Translational Meta-analytical Methods to Localize the Regulatory Patterns of Neurological Disorders in the Human Brain. 2015. AMIA Annu Symp Proc. 2073-82.

<b>David MM</b>, Mesle M, Malandain C, Cohen D, Vigier B, Barbier P, Lantzy R, Vogel TM. 2008. Molecular biology-based strategy for site remediation. ISBN 1-57477-163-9, published by Battelle, Columbus, OH.

<b>David MM</b>, Simonet P, Vogel TM. DNA Shuffling: a dynamic model for novel gene synthesis. 2007. International Journal of Astrobiology. 6:59–87.
